openhtf/io/output/mfg_inspector.py

```diff
@@ -252,10 +252,10 @@ def _ExtractParameters(record, testrun, used_parameter_names):
         testrun_code = testrun.failure_codes.add()
         testrun_code.code = testrun_param.name
         if measurement.dimensions is None:
-          if testrun_param.numeric_value:
-            testrun_code.details = str(testrun_param.numeric_value)
-          else:
+          if testrun_param.text_value:
```

Comment:
I tried this initially.  For the string case numeric_value is still populated with 0 and then enters that block.

I'll change it to check for float instead since we cast above for numeric case